tee-worker/litentry/core/data-providers/src/achainable.rs

```diff
@@ -255,7 +256,7 @@ impl TryFrom<AchainableParams> for Params {
 				let network = &p.chain;
 				let amount = ap.to_string(&p.amount)?;
 
-				let p = ParamsBasicTypeWithAmount::new(name, network, amount);
+				let p = ParamsBasicTypeWithAmount::new(name, &network[0], amount);
```

Comment:
@zhouhuitian @Kailai-Wang in the (existing) `AmountToken` case we're just using the first chain in the vector; is that a reasonable solution for the rest of the assertions, or should we make a more significant fix?